src/url-loader/package-url-resolver.ts

```diff
@@ -27,6 +27,7 @@ export interface PackageUrlResolverOptions {
   packageDir?: string;
   componentDir?: string;
   host?: string;
+  protocol?: string;
```

Comment:
I think we should document the meaning of these params somewhere. Since this is the most common url resolver, this might be the right place?

Definitely either here or in FS url resolver though.